util/slack.q

```diff
@@ -21,11 +21,13 @@ taguser:{[uid] "<@",uid,">"}
 /-- api calls --
 /the following are wrappers around slack API calls (of the same name) exposing very basic functionality
 /for more advanced usage of these API calls, more sophisticated code will be necessary
-channels.list:{.Q.hp[url`channels.list;.post.ty`form;.post.urlencode (1#`token)!enlist token]}
-channels.info:{[chid].Q.hp[url`channels.info;.post.ty`form;.post.urlencode `token`channel!(token;chid)]}
+conversations.list:{.Q.hp[url`conversations.list;.post.ty`form;.post.urlencode (`token`limit`exclude_archived)!(token;500;"true")]}
+conversations.info:{[chid].Q.hp[url`conversations.info;.post.ty`form;.post.urlencode `token`channel!(token;chid)]}
+channels.list:conversations.list
+channels.info:conversations.info
 chat.postMessage:{[chid;msg].Q.hp[url`chat.postMessage;.post.ty`form;.post.urlencode `token`channel`text!(token;chid;msg)]}
-groups.list:{.Q.hp[url`groups.list;.post.ty`form;.post.urlencode (1#`token)!enlist token]}
-groups.info:{[chid].Q.hp[url`groups.info;.post.ty`form;.post.urlencode `token`channel!(token;chid)]}
+groups.list:conversations.list
```

Comment:
Don't think this is correct - based on a quick look at Slack API docs, either this should probably be a separate request with `types=private_channel`, or previous request should have `types=public_channel,private_channel`. In old API, "groups" were private channels (as far as I can remember, been a while since I wrote or touched this code)

Reply:
Looks like im.list is also deprecated, will work in a fix for that too

Reply:
Updated to a nicer version where all the old calls now call the new version with specific arguments. It should be OK to force the new style API calls to supply a list of types